lib/phx_tools_web/router.ex

```diff
@@ -14,8 +14,12 @@ defmodule PhxToolsWeb.Router do
     plug PhxToolsWeb.SystemDetector
   end
 
+  pipeline :curl_detector do
+    plug PhxToolsWeb.CurlDetector
+  end
+
   scope "/", PhxToolsWeb do
-    pipe_through [:browser, :system_detector]
+    pipe_through [:browser, :system_detector, :curl_detector]
```

Comment:
Change the order. We don't need `:system_detector` for `:curl_detector` to work.